src/client/components/ModalWithFeedback/ModalWithFeedback.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+import './ModalWithFeedback.css';
+import Input from '../Input/Input';
+import { EventCardText } from '../EventCardText/EventCardText';
+import Modal from '../Modal/Modal';
+
+function ModalWithFeedback(props) {
+  const [text, textName] = useState('');
```

Comment:
It's common to name the second element `setText` since it sets the previous element.
For example: `const [value, setValue] = useState(defaultValue);`